content/v0.1.markdown

```diff
@@ -0,0 +1,122 @@
+is designed in a modular bottom-up fashion: we currently have several independant low-level modules,
+each managing a very specific part of what "being a wayland compositor" requires. Smithay's user can
+then use these modules and configure them as they please to set up their compositor.
+
+We will consider integrating higher-level modules into Smithay in the future. They would be built on
+top of the low-level ones, but there is still much work to do with the low-level ones, and this
+takes priority!
+
+Smithay modules are currently organized in two groups: backend modules and wayland modules.
+
+#### Backend modules
+
+The backend modules are here to ease integration with the operating system. They abstract away the
+setup of the framebuffers, initialization of an OpenGL context to draw on, and retrieval of user
+inputs. Currently two backends exist:
+
+- The DRM + libinput backend, which allows to start a smithay-based compositor directly in a TTY,
+  like a regular graphical environment.
+- the winit backend, which allows to start a smithay-based compositor as a winit app (and as such
+  as an X11 or wayland client). Being able to launch the compositor as a client helps a lot with
+  the development and debugging.
+
+#### Frontend modules
+
+The wayland modules take care of several aspects of the wayland protocol and manage the
+state of the wayland clients. Currently 3 main tasks are implemented:
+
+- Keeping track of shared memory buffers sent by the clients in the form of a shared memory map
+  (note: this does not handle opengl clients).
+- Keeping track of the client surfaces and their relations to each other. Wayland clients define their
+  window contents by creating one or more surfaces, positioning them relative to each other, and
+  attaching buffers to them.
+- Forwarding inputs to the clients selectively based on current pointer or keyboard focus.
+
+All this together already allows the handling of several type of clients. For example, it is possible to
+run `weston-terminal` (a wayland terminal emulator provided by weston) in the smithay examples:
+
+![Smithay running weston-terminal](/images/smithay-0-1.png)
+
+*The black borders are a graphical artefact from smithay's example, that we just haven't got around
+fixing yet.*
+
+## How can I run it?
+
+If you want to play around with smithay and see what it can do, check out the
+[examples in the github repository](https://github.com/Smithay/smithay/tree/master/examples). A lot
+of code is still needed for a compositor built on smithay to work, and there is not nearly enough
+space in this blog post to paste the code.
+
+You can also check out [Smithay's documentation on docs.rs](https://docs.rs/smithay). The crate-level
+docs are minimalistic, but we try to expand on all the details in the documentation of the individual
+modules.
+
+## What's next?
+
+For the 0.2 target, we plan to keep workin on the fundamentals. Some of the most important remaining features are:
```

Comment:
"(and possibly 0.3)" maybe?

Reply:
Yeah, you're right, maybe I've been overloading the 0.2 milestone